config/default_records/request_types/012_limber_scrna_core_cell_extraction.yml

```diff
@@ -0,0 +1,14 @@
+# These purposes are associated with the Limber scRNA Core Cell Extraction pipeline.
+# They are in a separate file to allow us to soft-release them before
+# the real release.
+---
+limber_scrna_core_cell_extraction:
+  name: scRNA Core Cell Extraction
+  asset_type: SampleTube
+  order: 1
+  request_class_name: CustomerRequest
+  for_multiplexing: false
+  billable: true
+  product_line_name: scRNA Core Cell Extraction
```

Comment:
I don't think we should be creating a specific Product Line for this pipeline. From the product_line.rb model:
```
# ProductLine represents the team doing the work. Used primarily to group together
# {SubmissionTemplate submission templates} for display, but is also used in downstream
# reporting.
#
# @note This value is becoming increasingly unreliable as the teams become more adaptable
#       with sharing workload. It is probably not a good idea to base critical behaviour off this.
```

I think to add to the note above, it's also becoming increasingly unreliable as we haven't known what it's for in the past so have just been making new ones for new pipelines 😞 

I will post in Slack asking what the correct one to use is - think from looking at the list it might be 'Illumina-Htp', but not sure. The values we have probably also don't represent the actual teams very well either.